modules/videoio/src/cap_v4l.cpp

```diff
@@ -1765,8 +1770,12 @@ static int icvSetPropertyCAM_V4L( CvCaptureCAM_V4L* capture,
             retval = v4l2_reset(capture);
             width = height = 0;
         }
-        break;
+    }
+    break;
     case CV_CAP_PROP_FRAME_HEIGHT:
+    {
+        int& width = capture->width_set;
+        int& height = capture->height;
```

Comment:
`capture->height`  => `capture->height_set` ?

Reply:
corrected. sorry for the mistake when preparing the PR patch.